src/Maestro/DependencyUpdateErrorProcessor/.config/settings.json

```diff
@@ -5,5 +5,6 @@
   "GitHub": {
     "GitHubAppId": "[vault(github-application-id)]",
     "PrivateKey": "[vault(github-application-private-key)]"
-  }
+  },
+  "EnableDependencyUpdateErrorProcessor": "[config(FeatureManagement:DependencyUpdateErrorProcessor)]"
```

Comment:
Same with this change? Is it intentional?

Reply:
Yes , Alex made some changes to the fetch the feature flag directly from the config. So I redid this part.  Should I add a small part saying I changed this in the issue?